presets.js

```diff
@@ -9,8 +9,7 @@ export function getPresets() {
 
 	let presets = {}
 
-	if (this.switch.poePortConfig) {
```

Comment:
The conditional isn't needed here because `self.poe_status` is guaranteed to be present when this code runs, but I didn't re-indent the code here to make the changes more obvious.

I tried running `prettier` on it but the result was a bit of a mess – this can be restructured to make prettier a bit happier but I don't think it makes sense in this PR. Same deal with feedbacks.